ui/app/components/ui/searchable-item-list/list-item-search/list-item-search.component.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect, useRef } from 'react'
+import PropTypes from 'prop-types'
+import Fuse from 'fuse.js'
+import InputAdornment from '@material-ui/core/InputAdornment'
+import TextField from '../../text-field'
+
+const generateFuseKeys = (keys) => {
+  const weight = 1 / keys.length
+  return keys.map((key) => ({ name: key, weight }))
+}
+
+function usePrevious (value) {
+  const ref = useRef()
+  useEffect(() => {
+    ref.current = value
+  })
```

Comment:
This is missing the `value` dependency:

```suggestion
  }, [value])
```

Reply:
https://github.com/MetaMask/metamask-extension/pull/8877/commits/349280d1070a3be1c6ef2d8994514dc1f782c941